docker/entrypoint.sh

```diff
@@ -1,6 +1,26 @@
 #!/bin/bash
 
-kolibri manage migrate
-kolibri services --background
+if [ $RUN_BG_SERVICE = 'true' ]
```

Comment:
Nit: same as the other env var. Is there any case where entrypoint.sh is called without RUN_BG_SERVICE being true?